apps/demos/src/app.scss

```diff
@@ -0,0 +1,30 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+}
+
+// replace this with nav component as soon as it is ready - if there is one
```

Comment:
There is a nav component. Can't we use that one?

Reply:
It currently not capable of expanding/collapsing